### PR TITLE
Add "Run in CI" documentation

### PR DESCRIPTION
**Pull request by flaurida**
* Also remove dead links on home page where we don't have docs yet

### CODE REVIEW

File: docs/run_in_ci.md [hunk 1]
```diff
@@ -3,4 +3,87 @@ id: run_in_ci
...
+
+You can also specify environment variables, ports, volumes, and options for your service. See [GitHub's documentation on services](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idservices) for a more detailed explanation.
+
+## GitHub Actions Documentation
+
+To learn more about configuring workflows in GitHub Actions, see [GitHub's documentation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-workflows).
+
+## Debugging Failures
+
+Each test generates debugging artifacts that you can use to understand failures. These include a video, gif, interactive DOM recording, and browser logs.
```
Comment (jperl) [inline docs/run_in_ci.md]: include gif below here?

Comment (flaurida) [reply docs/run_in_ci.md]: I want to actually link to the debugging doc which should have it. that ok? we should write it tmrw

Comment (jperl) [reply docs/run_in_ci.md]: We can add the link when that is written

Comment (jperl) [reply docs/run_in_ci.md]: ok

Comment (flaurida) [reply docs/run_in_ci.md]: Yep, as we write new docs we should link to them from current docs

**Pull request merged**